content/blog/2021/12/2021-12-03-election-results.adoc

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "2021 Jenkins Board and Officer Elections Results"
+tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- halkeye
+opengraph:
+  image: /images/governance/elections/2021/opengraph.png
+description: >
+  This blogpost summarizes the election results
+---
+
+image:/images/governance/elections/2021/opengraph.png[Jenkins Elections, role=center, float=right, height=178]
+
+The Jenkins community has recently completed the 2021 elections.
+On behalf of the Jenkins community and the elections committee,
+we congratulate all newly elected board members and officers!
+We also thank all candidates and voters who participated this year.
+
+Election results:
+
+* link:https://github.com/MarkEWaite[Mark Waite] joins the re-elected link:https://github.com/oleg-nenashev[Oleg Nenashev] and  
```

Comment:
Can you think of good phrasing? "Because oleg has left cloudbees, there is no longer a corporate of restriction of having both mark and oleg?"

Reply:
I added some phrasing in e8872702e931c5f06804910c09a430781967c2a5.  @uhafner does that phrasing work for you?  I'd like to publish this within the next hour or two so that we can promote it on social media before end of day European time.

Reply:
👍